specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/textanalytics.json

```diff
@@ -378,63 +674,360 @@
       },
       "allOf": [
         {
-          "$ref": "#/definitions/AnalyzeTextTask"
+          "$ref": "#/definitions/AnalyzeTextTask"
+        }
+      ],
+      "x-ms-discriminator-value": "PiiEntityRecognition"
+    },
+    "AnalyzeTextLanguageDetectionInput": {
```

Comment:
We don't support LD in analyze. What is this for?

Reply:
correct we do not support. also sentiment or any other non conversation task.
I think we just took the TA as basis and added on top of it. Do we want a separate swagger for conversation?

Reply:
This might have been from the 03-01-2022 API version which was still in review when we started the API. I will merge in the latest changes in case this is no longer relevant.

Reply:
n/a